deploy/3-configure-feeds.ts

```diff
@@ -45,4 +44,4 @@ const func: DeployFunction = async function ({ network, deployments, getNamedAcc
 
 export default func;
 func.tags = ["configure"];
-func.skip = async env => env.network.live;
+func.skip = async env => !env.network.live;
```

Comment:
I think this script is only valid for hardhat, not for live networks. We'll use VIP to configure the price feeds on mainnets

Reply:
We could (some alternatives):
* define assets for hardhat
* reuse the assets from another network (as we made with `bsctestnet`)
* remove this deployment script

Reply:
On hardhat we'll generally want to use mock oracles to be able to simulate price changes easily. We can reuse testnet assets as long as the we fetch the contract to get the address

Reply:
Ok, let's remove this script then